props/prop_base.h

```diff
@@ -603,9 +607,16 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   bad_blade:
     STDOUT.println("BAD BLADE");
 #ifdef ENABLE_AUDIO
+#ifdef DISABLE_TALKIE
```

Comment:
These needs to be moved to a common location, and have comments that describe what they sound like.

Reply:
I assume you mean a comment explaining what the define does and the beeps that replace the messages?  Where would you recommend?

I had attempted to centralize all errors to a single file but it would not compile so I left the change to beeper in the existing location.